recipes/soci/all/conandata.yml

```diff
@@ -0,0 +1,8 @@
+sources:
+  "4.0.1":
+    url: "https://github.com/SOCI/soci/archive/4.0.1.tar.gz"
+    sha256: "fa69347b1a1ef74450c0382b665a67bd6777cc7005bbe09726479625bcf1e29c"
+  "4.0.0":
+    url: "https://github.com/SOCI/soci/archive/4.0.0.tar.gz"
+    sha256: "359b988d8cbe81357835317821919f7e270c0705e41951a92ac1627cb9fe8faf"
+
```

Comment:
There's no need to add this as no one will end up using it since we have a newer version

```suggestion
```

Reply:
Version 4.0.0 is the current stable release, so it should be included. Lower versions are not included since they seem not to be supported by the current cmake build system.

Reply:
I am not sure what you mean by "current stable release" I am assuming this follows semantic version and the 4.0.1 release i a patch (fix) over that 4.0.0

If you are adding a newer release, please remove the older one.